src/mcts/node.h

```diff
@@ -97,8 +96,8 @@ class Edge {
   Move move_;
 
   // Probability that this move will be made. From policy head of the neural
-  // network.
-  float p_ = 0.0;
+  // network. Stored with a 16 bit significand in the form of a uint16_t.
```

Comment:
Stored in 16-bit compressed format (5 bits of exponent and 11 bits of mantissa).